Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorWidget.cpp

```diff
@@ -857,6 +861,8 @@ namespace AzToolsFramework
 
                 m_sourceAssetId = assetId;
             }
+
+            AssetEditorNotificationsBus::Event(m_observerId, &AssetEditorNotifications::OnAssetCreated, assetId);
```

Comment:
This event should be sent by the `CreateAsset` function above and not the OncatalogAssetAdded function.

Reply:
That won't work as the CreateAsset function is called when the window is opened and a placeholder assetId is in use. We need to wait until the asset is saved and loaded so we know the correct assetId (and name). You're correct that it will trigger on irrelevant assets though: I've moved the bus call to inside the type and path check, which should correct the problem.

Reply:
The relative path and AssetType ID validation check is good enough to ensure that the event is only sent for the specific observer.
This commit is good with me now.